### ISSUE TITLE
instead of starting the interface, it just says: Done! Press any key to continue . . . and then closes! 

### ISSUE THREAD

**Issue by sgpascoe** [oobabooga/text-generation-webui#2333]
### Describe the bug

bin D:\ai\oobabooga_windows\installer_files\env\lib\site-packages\bitsandbytes\libbitsandbytes_cuda117_nocublaslt.dll
INFO:Loading TheBloke_manticore-13b-chat-pyg-GPTQ...
INFO:Found the following quantized model: models\TheBloke_manticore-13b-chat-pyg-GPTQ\Manticore-13B-Chat-Pyg-GPTQ-4bit-128g.no-act-order.safetensors

Done!
Press any key to continue . . .

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Reproduction

I run windows.bat

### Logs

```shell
bin D:\ai\oobabooga_windows\installer_files\env\lib\site-packages\bitsandbytes\libbitsandbytes_cuda117_nocublaslt.dll
INFO:Loading TheBloke_manticore-13b-chat-pyg-GPTQ...
INFO:Found the following quantized model: models\TheBloke_manticore-13b-chat-pyg-GPTQ\Manticore-13B-Chat-Pyg-GPTQ-4bit-128g.no-act-order.safetensors

Done!
Press any key to continue . . .
```

### System Info

```shell
windows 10
nvidia 1080ti
```

**Comment by ozzeruk** [comment]
This happens to me if it doesn't properly recognise the type from the filename. Perhaps try changing the directory name in case that has confused it. It happened to me when I renamed the model file, turns out it's using the file name to deduce default settings.

**Comment by ArkyonVeil** [comment]
Suffering from a very similar issue, attempting to debug and finding a solution.

**Comment by toby3d** [comment]
Similar behavior on Arch Linux. After the upgrade, all previously working llama models stopped working without visible errors, even with the `--verbose` flag. Other combinations of flags don'T help either. Reinstalling text-generation-webui completely didn't change anything.

**Comment by Johnnycyan** [comment]
Same issue. Happens with TheBloke_WizardLM-30B-Uncensored-GPTQ and Zicara_OPT-30B-Erebus-4bit-128g

**Comment by toby3d** [comment]
I rewound the commit history all the way back to the beginning of April, hoping to find one before which the models would load as needed. But the behavior persisted: a long wait with a soft shutdown. Which is strange, because before the update yesterday I had a version from a week ago in which the models worked fine. It would make sense that I wouldn't need to rewind to a month ago if the problem was in the project.

I have a suspicion that the problem lies in the dependencies, that llama model support was broken in some of them. In rewinding the history of the project, I have not dealt with reinstalling or updating the dependencies, keeping them as they were yesterday.

Will try to rewind again, this time with reinstalling the dependencies to find the commit after which llama was broken.

**Comment by jfjensen** [comment]
After installation, I got the web interface to start up once. And it worked just fine.
Now, I cannot get the interface to start anymore. I get the same "error" as mentioned in the original post but with the "mayaeary_pygmalion-6b_dev-4bit-128g" model (safetensor).

**Comment by topcheese** [comment]
I'm having the same issues with "TheBloke_wizard-mega-13B-GPTQ."  I'm on Win 10 Build 19045.2965.  I'm think my problem maybe because I chose the CPU option on my laptop.

**Comment by ozzeruk** [comment]
> I'm having the same issues with "TheBloke_wizard-mega-13B-GPTQ." I'm on Win 10 Build 19045.2965. I'm think my problem maybe because I chose the CPU option on my laptop.

Yeah I think if you choose the CPU option then you should pick a GGML model, whether you end up using GPU acceleration or not. That option should probably be renamed "llama.cpp backend".

**Comment by topcheese** [comment]
@ozzeruk I suspected it may have been the model and the CPU option I selected.  I missed the part where it suggested the proper models to load.  Thanks for the info!

**Comment by ohh25** [comment]
The webui starts, but loading the model will cause _Press any key to continue . . ._ ,even after fully reinstalled.
And it seems almost all wizardlm models can't load for me. form latest WizardLM-Uncensored-SuperCOT-StoryTelling-30B-GPTQ to Wizard-Vicuna-13B-Uncensored-GPTQ

**Comment by ArkyonVeil** [comment]
Greetings fine folks. I've been trying to get this fixed here and then and finally managed to find a solution:
---
By changing the lines as listed here in this comment, the model finally loads: https://github.com/oobabooga/text-generation-webui/issues/1066#issuecomment-1504644213

File: modules/GPTQ_loader.py - line 63
---
Disclaimer: Do note I've also updated oobabooga to the latest, increased my page file to over 50 gb, an am using the latest WizardLM Model from https://huggingface.co/TheBloke/WizardLM-30B-GPTQ
Despite these changes, the GPTQ_loader.py tweak is the only thing that worked, though given the nature of these things YMMV
